pkgs/development/python-modules/pytzdata/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  version = "2017.3.1";
+  pname = "pytzdata";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wi3jh39zsa9iiyyhynhj7w5b2p9wdyd0ppavpsrmf3wxvr7cwz8";
+  };
+
+  # No tests
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Official timezone database for Python";
```

Comment:
This is the description on its website, but I do not see anything official about it. Could you change it to "Timezone database for Python"?

It is quite unfortunate that it ships an outdated version of tzdata rather than using system tzdata, but this is out of scope of this PR.